Assets/Script/Player_control.cs

```diff
@@ -18,16 +18,16 @@ void Update()
     {
         if (Input.GetKeyDown(toggleFront))
         {
-            ToggleFront();
+           /* ToggleFront();
```

Comment:
Should we keep this Player_control.cs class? @Kam0ss

Reply:
Since we keep the flycam, yes for now, but the commented code can be deleted since it's not useful anymore.